src/functionalTest/java/uk/gov/hmcts/reform/ccd/document/am/befta/DataStoreTestAutomationAdapter.java

```diff
@@ -0,0 +1,126 @@
+package uk.gov.hmcts.reform.ccd.document.am.befta;
+
+import io.restassured.RestAssured;
+import io.restassured.builder.RequestSpecBuilder;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.gov.hmcts.befta.BeftaMain;
+import uk.gov.hmcts.befta.DefaultTestAutomationAdapter;
+import uk.gov.hmcts.befta.data.UserData;
+import uk.gov.hmcts.befta.exception.FunctionalTestException;
+
+import java.io.File;
+import java.util.HashMap;
+import java.util.Map;
+
+public class DataStoreTestAutomationAdapter extends DefaultTestAutomationAdapter {
```

Comment:
class name needs adjustments as per this application.

Reply:
Updated.  CaseDocumentAmTestAutomationAdapter